src/services/api-service.ts

```diff
@@ -49,7 +56,7 @@ export default abstract class APIService {
       return { content, ...wrapper }
     }
 
-    return wrapper
+    return this.returnErroneousReponse(wrapper, responseBody)
```

Comment:
🤣🤣a bit on the poetic side, isn't it? 🤣🤣🤣 Not sure about using "return" in a function name. I'd prefer getErrorResponse()

Reply:
Fixed in #61